editions/tw5.com/tiddlers/widgets/EditWidget.tid

```diff
@@ -24,3 +24,16 @@ The content of the `<$edit>` widget is ignored.
 |inputActions |<<.from-version 5.1.23>> Optional actions that are triggered every time an input event occurs within the input field or textarea |
 |refreshTitle |<<.from-version 5.1.23>> An optional tiddler title that makes the input field update whenever the specified tiddler changes |
 
+! Examples
+
+!! Edit the contents (text field) of a tiddler titled with today’s date
+
+<$macrocall $name=".example" n="1"
+eg="""<$edit tiddler=<<now YYYY-0MM-0DD>> class="tc-edit-texteditor"/>
+"""/>
```

Comment:
@springerspandrel , nice examples! I prefer the new testcase widget for examples. Especially since this tiddler is a blank canvas as far as examples go.  This example could be replaced with testcase widget something like this:

```
<$testcase>
<$data title=Description text="Simple $edit widget"/>
<$data title=Narrative text="""Edit the contents (text field) of a tiddler titled with today’s date"""/>
<$data title=Output text="""<$edit tiddler=<<now YYYY-0MM-0DD>> class="tc-edit-texteditor"/>"""/>
</$testcase>
```

Testcase widget is interactively editable. And it is very helpful when the examples create tiddlers as it is very easy to see the tiddler get created (new tab appears in the test case).

What do you think?

Reply:
Thanks @btheado 

@springerspandrel I think it makes sense to move towards using the `testcase` widget, but I'm happy to merge this as is, and happy to accept further updates.